### DISCUSSION TITLE
List of prominent blog posts

### DISCUSSION THREAD

**Discussion by phated** [gulpjs/gulp#978] [Show and tell]
One of our goals for the 4.0 release is to email prominent blog posts and ask them to update with the new syntax.  So my question is, what are the prominent blog posts? @contra @travm @sindresorhus

**Comment by ilanbiala** [comment]
@alexgorbatchev has a couple Gulp posts. Here are some other ones: 
https://scotch.io/tutorials/automate-your-tasks-easily-with-gulp-js
http://leveluptuts.com/tutorials/learning-gulp
http://code.tutsplus.com/tutorials/managing-your-build-tasks-with-gulpjs--net-36910
http://www.sitepoint.com/introduction-gulp-js/
http://www.smashingmagazine.com/2014/06/11/building-with-gulp/

**Comment by wethecode** [comment]
maybe this series: http://stefanimhoff.de/2014/gulp-tutorial-1-intro-setup/

**Comment by strugee** [comment]
There's also https://github.com/markdalgleish/presentation-build-wars-gulp-vs-grunt which, while not having a lot of stars, is the first result for a search for "gulp vs. grunt".

**Comment by travm** [comment]
My articles are updated and ready to go live as soon as 4.0 hits. I'll keep an eye out for other articles and see if I can get in touch with the authors to make sure they are aware of the changes.

**Comment by pikeas** [comment]
@travm Why not post those articles now? Gulp 4 will drop soon enough (@phated any ETA?), and many people will want to get a headstart on their migrations.

**Comment by yocontra** [comment]
@pikeas https://github.com/gulpjs/gulp/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22gulp+4%22

**Comment by hamstu** [comment]
I remember using @greypants' gulp starter.
- Article: https://www.viget.com/articles/gulp-browserify-starter-faq
- Repo: https://github.com/vigetlabs/gulp-starter

**Comment by JohnMunsch** [comment]
It might be worth hitting some of the most prominent Gulp questions on Stack Overflow dealing with things Gulp v4 makes easier or fixes (for example):
- http://stackoverflow.com/questions/22824546/how-to-run-gulp-tasks-synchronously-one-after-the-other/22826429#22826429
- http://stackoverflow.com/questions/22391527/gulps-gulp-watch-not-triggered-for-new-or-deleted-files?s=34|0.0000

**Comment by ghost** [comment]
I am totally on board with @ilanbiala suggestion of leveluptuts. Scotts [video series on youtube](https://youtu.be/wNlEK8qrb0M?list=PLLnpHn493BHE2RsdyUNpbiVn-cfuV7Fos) was what got me started with gulp 3 all those years ago, time for a refresh with gulp 4  :+1:
